openmdao/visualization/meta_model_viewer/meta_model_visualization.py

```diff
@@ -0,0 +1,743 @@
+"""Define output of Meta Models and visualize the results."""
+
+from collections import OrderedDict
+import math
```

Comment:
We try to stick to pep-8 when organizing our imports. Here, the python ones should be first, then general packages like numpy/scipy, then probalby the bokeh stuff, and finally the openmdao ones.

https://www.python.org/dev/peps/pep-0008/#imports